props/prop_base.h

```diff
@@ -956,7 +957,7 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
   }
 
   // The prop should call this from Loop() if it wants to detect twists.
-  void DetectTwist() {
+  virtual void DetectTwist() {
```

Comment:
Since prop_base never calls this function, there is no need to make it virtual.

Reply:
done.